lib/engines/content_block_manager/app/workers/content_block_manager/delete_draft_content_blocks_worker.rb

```diff
@@ -0,0 +1,17 @@
+require "sidekiq/api"
+
+module ContentBlockManager
+  class DeleteDraftContentBlocksWorker < WorkerBase
+    sidekiq_options queue: :content_block_publishing
+
+    def perform
+      draft_content_editions = ContentBlockManager::ContentBlock::Edition.where(state: "draft").where("created_at < ?", 60.days.ago)
+      puts "Starting to delete #{draft_content_editions.count} draft Content Blocks"
+      draft_content_editions.each do |draft_content_edition|
+        puts "Deleting draft Content Block Edition #{draft_content_edition.id}"
+        ContentBlockManager::DeleteEditionService.new.call(draft_content_edition)
```

Comment:
I did wonder if we could amend `DeleteEditionService` to handle deleting editions in bulk, but I don't think it's really worth doing, as we'll likely only be dealing with a handful of editions at a time.

Reply:
Maybe limit it to 100?

Reply:
cool, I've added `limit` to the end of the query... but testing it by creating 101 records in a unit test is really slow :0 any ideas how that might be speeded up?

Reply:
I don't think it's worth creating all those records. We should trust that ActiveRecord does its thing correctly.